e2e/test/scenarios/permissions/sandboxing/sandboxing-misconfiguration.cy.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { USER_GROUPS, WRITABLE_DB_ID } from "e2e/support/cypress_data";
+
+import {
+  assertResponseFailsClosed,
+  assignAttributeToUser,
+  configureSandboxPolicy,
+  getCardResponses,
+  gizmoViewer,
+  rowsShouldContainOnlyOneCategory,
+  signInAs,
+} from "./helpers/e2e-sandboxing-helpers";
+
+const { H } = cy;
+
+describe("admin > permissions > sandboxing > misconfiguration", () => {
+  before(() => {
+    H.restore("postgres-writable");
+
+    cy.signInAsAdmin();
+    H.setTokenFeatures("all");
+
+    H.blockUserGroupPermissions(USER_GROUPS.ALL_USERS_GROUP, WRITABLE_DB_ID);
+    H.blockUserGroupPermissions(USER_GROUPS.COLLECTION_GROUP, WRITABLE_DB_ID);
+    H.blockUserGroupPermissions(USER_GROUPS.READONLY_GROUP, WRITABLE_DB_ID);
+
+    // @ts-expect-error - this isn't typed yet
+    cy.createUserFromRawData(gizmoViewer);
+
+    cy.log("Create a simple editable products table");
+    H.queryWritableDB("DROP TABLE IF EXISTS products");
+    H.queryWritableDB(
+      "CREATE TABLE IF NOT EXISTS products (id INT PRIMARY KEY, category VARCHAR, name VARCHAR)",
+    );
+    H.queryWritableDB(
+      "INSERT INTO products (id, name, category) VALUES (1, 'A', 'Gizmo'), (2, 'B', 'Widget')",
+    );
+    H.resyncDatabase({ dbId: WRITABLE_DB_ID, tableName: "products" });
```

Comment:
since you're altering this table in the body of the test, I'd recommend putting this setup in a beforeEach hook. also consider using `resetTestTable`